### ISSUE TITLE
Function documentation

### ISSUE THREAD

**Issue by PonteIneptique** [charignon/hooklib#1]
Hey,
Looks like a great lib to me. But to be honnest, not sure anyone could make use of it without  function doc.

**Comment by PonteIneptique** [comment]
I'd be really happy to use it and collaborate if this lib is what i think it is.

**Comment by charignon** [comment]
I agree, the examples are really not enough to make a real use of it.
Looking at the integration code, I am pretty sure people can figure it out but it is not very user friendly.
That's a great suggestion, I will add documentation both to the readme and in the code.
Feel free to collaborate, I would love that!

**Comment by charignon** [comment]
This should be good now, you can have a look at the readme and let me know what you think.

**Comment by PonteIneptique** [comment]
I think that's better. Through, what I'd expect from a library which I'd really want to use is function documentation, using some standards.

I personally use the Sphinx doc standard ( http://sphinx-doc.org ). You could find example in one of my repo, such as https://github.com/Capitains/MyCapytain/blob/master/MyCapytain/endpoints/cts5.py ?

The good think with it is that you can then generate full docs such as http://mycapytain.readthedocs.org/ :)
